controllers/controllers.js

```diff
@@ -4,6 +4,30 @@ const jwt = require("jsonwebtoken");
 const config = require("../config/index");
 const Pet = require("../models/Pet");
 
+const createUser = async (req, res, next) => {
+  try {
+    let newUser;
+    console.log(req.body);
+    if (req.body.role === "user" || req.body === "admin") {
+      newUser = await new User(req.body);
+      await newUser.save();
```

Comment:
You can use outside of this two conditions, 
`
if () { }
else { }

await newUser.save()
res.status(201).json(newUser);
`

Reply:
Thanks, refactored code!